Makefile

```diff
@@ -51,6 +51,12 @@ install_cargo_nextest: install_rs_build_toolchain
 	cargo $(CARGO_RS_BUILD_TOOLCHAIN) install cargo-nextest --locked || \
 	( echo "Unable to install cargo nextest, unknown error." && exit 1 )
 
+.PHONY: install_cbindgen # Install cbindgen to generate the C API header
+install_cbindgen: install_rs_check_toolchain
+	@cbingen --version > /dev/null 2>&1 || \
+	cargo $(CARGO_RS_BUILD_TOOLCHAIN) install cbindgen --locked || \
+	( echo "Unable to install cbindgen, unknown error." && exit 1 )
```

Comment:
here you do a install_rs_check_toolchain but use the build toolchain

Reply:
added

Reply:
there still is a mismatch between the toolchain installed and the one used

Reply:
😨